coverage/.resultset.json

```diff
@@ -1,15 +1,39 @@
 {
   "RSpec": {
     "coverage": {
-      "/Users/omidafshar/Documents/CS169/Grocer-Plus/spec/hi_spec.rb": [
+      "/Users/babajoon/nowtmp/Grocer-Plus/spec/recipe_spec.rb": [
```

Comment:
These are files from personal machines. This should be added to a .gitignore

Reply:
Ok, will do!